src/components/atoms/RadioButton/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+export type RadioButtonProps = {
+  id?: string;
+  value?: string;
+  onChange?: () => void;
+  checked?: boolean;
+  label?: string;
+  name?: string;
+};
+
+const RadioButton: React.FC<RadioButtonProps> = ({
+  id,
+  onChange,
+  value,
+  checked,
+  name,
+}) => {
+  return (
+    <div className="m-1 h-[22px] w-[22px] rounded-full">
+      <label htmlFor={id} className="relative h-2 w-2 cursor-pointer">
```

Comment:
Move cursor-pointer to the parent so we can have a wider click area;

![image](https://user-images.githubusercontent.com/8492847/209441962-233525d2-b395-4a87-be43-b45a73ee6cde.png)

![image](https://user-images.githubusercontent.com/8492847/209441965-d0002c98-300b-4ad1-bab1-857c8e549d2c.png)